tests/test_parser.py

```diff
@@ -357,7 +357,7 @@ def test_comments_insert(self):
         )
 
         self.assertEqual(expression.comments, ["comment1.1", "comment1.2", "comment1.3"])
-        self.assertEqual(expression.this.comments, ["comment2"])
+        self.assertEqual(expression.this.this.comments, ["comment2"])
```

Comment:
why did this change? don't we push the comments up?

Reply:
Because this corresponds to a `Table`. May need to push the comments to tables as well.

Reply:
[done](https://github.com/tobymao/sqlglot/pull/3340/commits/47a57456d665a2ad65f5a80b0575f000abfcec43)